examples/asa/scripts/transfer/gold-contract-sc.js

```diff
@@ -22,6 +21,22 @@ async function run(runtimeEnv, deployer) {
 
   // Transaction FAIL as Elon tried to receive instead of John
   await transferAlgo(deployer, { addr: sender}, elonMuskAccount.addr, 50, lsig);
+
+  //group transactions
+  let transactions = [
+    mkTxnParams({addr: sender}, johnAccount.addr, 30, lsig, {}),
+    mkTxnParams({addr: sender}, johnAccount.addr, 40, lsig, {}),
+    mkTxnParams({addr: sender}, johnAccount.addr, 50, lsig, {})]
```

Comment:
can we add a normal transaction here? Eg sending Algo from `john` to `elon`?

Reply:
Ah, this will work only for lsig transfers. Please make a note about this somewhere here (in this file).

Reply:
I am removing this example from here.